x-pack/test/apm_api_integration/tests/alerts/rule_registry.ts

```diff
@@ -143,6 +152,12 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       };
 
       before(async () => {
+        try {
+          await es.indices.delete({
+            index: APM_METRIC_INDEX_NAME,
+          });
+          // eslint-disable-next-line no-empty
+        } catch (exc) {}
```

Comment:
same here, comment would be great

Reply:
Ah yeah I think I can undo this.

Reply:
Fixed here: 6c5e2bfaeb4bf6016ed016e37eb2c7cea7c6fce7